frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -119,25 +148,32 @@ const validationConfig: VMSettingsValidationConfig = {
           ]
         : [VMWizardProps.activeNamespace, VMWizardProps.virtualMachines];
     },
-    validator: validateVm,
+    validators: [validateVm],
   },
   [VMSettingsField.USER_TEMPLATE]: {
     detectValueChanges: [VMSettingsField.USER_TEMPLATE],
     detectCommonDataChanges: [VMWizardProps.userTemplates],
-    validator: validateUserTemplate,
+    validators: [validateUserTemplate],
   },
   [VMSettingsField.OPERATING_SYSTEM]: {
     detectValueChanges: [VMSettingsField.OPERATING_SYSTEM],
-    validator: validateOperatingSystem,
+    validators: [validateOperatingSystem],
   },
   [VMSettingsField.CPU]: {
     detectValueChanges: [VMSettingsField.CPU],
-    validator: asVMSettingsFieldValidator(validatePositiveInteger),
+    validators: [asVMSettingsFieldValidator(validatePositiveInteger)],
```

Comment:
do we need a cpu validation ?

Reply:
both validations for CPU and memory that check for a positive integer value seems unnecessary, as you cannot type anything that isn't a positive integer.
But I'm not sure what happens when you use a template that has an invalid value.
haven't tried it though, but I think we can address this question in another PR.
@yaacov  What do you think ?

Reply:
if the template has a vlidation:
``` json
{
    "name": "core-limits",
    "path": "jsonpath::.spec.domain.cpu.cores",
    "message": "cpu cores must be limited",
    "rule": "integer",
    "min": 100,
    "max": 800
}
```
and user enter 5 cpus, who checks that 5 is not valid value, what am I missing here ?

Reply:
@irosenzw we let the user enter any cpu numer they want, I think we should check for cpu outside the range ... is it problematic ?

For example: a "big-cpu-template" have a rule that min cpu is 18, and user entered 6, this is an invalid cpu number, do we check it ?

Reply:
First of all, the PR is for memory validation only at this point. In the future we will cover all the fields in the VM wizard.

Regarding your question, if this is the only validation, it will fail.
but it there is another template that has a validation of :
```
{
    "name": "core-limits",
    "path": "jsonpath::.spec.domain.cpu.cores",
    "message": "cpu cores must be limited",
    "rule": "integer",
    "min": 2
}
```
it will pass.